data/deadlines/02_finaid.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Financial Aid
+subtitle: Open until 21st May
+url: /finaid
+image: /images/cards/finaid1.jpeg
+---
+EuroPython is for everyone in our community, no matter your circumstances.
+Please don't hesitate to reach out if you need help and support to attend.
```

Comment:
I'd put a `mailto` link in place of `reach out`.

Reply:
But we want people to click in and read, no? Not sure how to put a mailto here.

Reply:
We don't want people to e-mail us, but to read the finaid page and fill out the form. In general, people requesting finaid should not (have to) e-mail us to apply for grants; they should only email us for other questions or comments that they may have.

We could change the text to something like this:

> Check out the Financial Aid page if you need help and support to attend.

Reply:
Changed in https://github.com/EuroPython/website/pull/235/commits/b71db1d19bdb620825b5d6127410267a2e97cf59 with thanks!